packages/site/cms-site/src/iframebridge/IFrameMessage.ts

```diff
@@ -69,34 +79,50 @@ export enum AdminMessageType {
     HoverComponent = "HoverComponent",
     ContentScope = "ContentScope",
 }
+
+/**
+ * The `IAdminBlockMessage` is sent from the admin to the site, whih block should be displayed in the iFrame.
+ */
 export interface IAdminBlockMessage {
     cometType: AdminMessageType.Block;
     data: {
         block: unknown;
     };
 }
 
+/**
+ * The `IAdminShowOnlyVisibleMessage` is sent from the admin to the site, when the "Show only visible" checkbox is toggled.
+ */
 export interface IAdminShowOnlyVisibleMessage {
     cometType: AdminMessageType.ShowOnlyVisible;
     data: {
         showOnlyVisible: boolean;
     };
 }
 
+/**
+ * The `IAdminSelectComponentMessage` is sent from the admin to the site, when a component is selected in the admin interface.
+ */
 export interface IAdminSelectComponentMessage {
     cometType: AdminMessageType.SelectComponent;
     data: {
         adminRoute: string;
     };
 }
 
+/**
+ * The `IAdminHoverComponentMessage` is sent from the admin to the site, when a component is hovered in the admin interface.
+ */
 export interface IAdminHoverComponentMessage {
     cometType: AdminMessageType.HoverComponent;
     data: {
-        adminRoute: string;
+        adminRoute: string | null;
```

Comment:
unsure about this change. But iFrameMessages are the contract between admin and site. those types should be the same.

Reply:
I'm not sure if the adminRoute can ever be null, so maybe it would be better to remove it in the admin?

Reply:
i think it can be null, when something gets "unhovered". I removed it first also in the admin, but then i thought typing adjustment would be better to add the null in the client.

Reply:
You're right, this was incorrect in the site, thanks!